augraphy/augmentations/inkbleed.py

```diff
@@ -18,6 +19,10 @@ class InkBleed(Augmentation):
     :param color_range: Pair of ints determining the range from which color
            noise is sampled.
     :type color_range: tuple, optional
+    :param kernel_size: Kernel size to determine area of inkbleed effect.
```

Comment:
I added `kernel_size` so that user is able to adjust the area of inkbleed effect.

For example, 7x7 kernel and 11x11 kernel:
![image](https://user-images.githubusercontent.com/52521979/147101009-aa8299aa-47a1-49f6-82f2-9f6e4d8a2cc7.png)